__tests__/integration/hathorwallet_facade.test.js

```diff
@@ -1970,6 +1970,46 @@ describe('mintTokens', () => {
     const tokenBalance4 = await hWallet.getBalance(tokenUid);
     const expectedAmount4 = expectedAmount2 + 100;
     expect(tokenBalance4[0]).toHaveProperty('balance.unlocked', expectedAmount4);
+
+    // Delegate mint back to wallet 1
+    const delegateResponse = await hWallet2.delegateAuthority(tokenUid, 'mint', address0);
+    expect(delegateResponse.hash).toBeDefined();
+    await waitForTxReceived(hWallet, delegateResponse.hash);
+    await waitForTxReceived(hWallet2, delegateResponse.hash);
```

Comment:
We need to start with the delegate because the last call sends the authority to `hWallet2`, so we need to send it back and continue the test